src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pdl/PdlPerson.kt

```diff
@@ -243,23 +247,30 @@ data class Vegadresse(
      */
     val UTM_GRENSE = 7_200_000
     val MINIMUM_AVSTAND_FOR_AUTOMATISK_BEREGNING_I_METER = 1000
-    fun fjerneBoforhold(annenVegadresse: Vegadresse?): Boolean {
+    fun fjerneBoforhold(annenVegadresse: Vegadresse?): AvstandTilSøkerDto {
         if (this.koordinater == null || annenVegadresse?.koordinater == null) {
-            return false
+            return AvstandTilSøkerDto(avstandIKm = null, langAvstandTilSøker = UKJENT)
         }
 
         val koordinater1 = this.koordinater
         val koordinater2 = annenVegadresse.koordinater
 
         if (koordinater1.x == null || koordinater1.y == null || koordinater2.x == null || koordinater2.y == null) {
-            return false
+            return AvstandTilSøkerDto(avstandIKm = null, langAvstandTilSøker = UKJENT)
         }
 
         if (koordinater1.y > UTM_GRENSE || koordinater2.y > UTM_GRENSE) {
-            return abs(koordinater2.y - koordinater1.y) > MINIMUM_AVSTAND_FOR_AUTOMATISK_BEREGNING_I_METER
+            val distanse = abs(koordinater2.y - koordinater1.y)
+            return AvstandTilSøkerDto(
+                avstandIKm = distanse.div(1000).toLong(),
+                langAvstandTilSøker = if (distanse > MINIMUM_AVSTAND_FOR_AUTOMATISK_BEREGNING_I_METER) JA_UPRESIS else UKJENT
```

Comment:
Litt usikker på denne delen
Skal vi vise avstandIKm for saksbehandleren i dette tilfelle? Som då viser ett tall som kanskje er helt feil når de søker opp avstanden mellaom søker og den andre personen?

Reply:
Pga at denne er "upresis" må vi i såfall vise denne avstanden på en annen måte enn om vi har gjort en realberegning